src/wolframite/base/package.clj

```diff
@@ -0,0 +1,49 @@
+(ns wolframite.base.package
+  "The place to be for loading and manipulating Wolfram packages.
+"
+  (:require
+   [clojure.string :as string]
+   [wolframite.core :as wl]
+   [wolframite.impl.wolfram-syms.intern :as wi]
+   [wolframite.wolfram :as w]))
+
+(defn load-symbols
```

Comment:
Should this be private or public? If public then when/how are users supposed to use it, and why would they prefer it over `<< / load-package!`?

Also, it is quite similar in name to wolframite.impl.wolfram-syms.wolfram-syms/load-all-symbols, so we should clearly communicate the different. Cognitive load is a b**ch!